### PR TITLE
Add warning on sshkey

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

#225

### CODE REVIEW

File: cmd/exo/cmd/ssh.go [hunk 1]
```diff
@@ -107,18 +109,19 @@ func getSSHInfo(name string, isIpv6 bool) (*sshInfo, error) {
 
 }
 
-func printSSHConnectSTR(info *sshInfo) error {
+func printSSHConnectSTR(info *sshInfo) {
 
 	if _, err := os.Stat(info.sshKeys); os.IsNotExist(err) {
-		return fmt.Errorf("default ssh KeyPair not found %q infos.sshKeys", info.sshKeys)
+		log.Println("Warning: Identity file Default ssh keypair not found not accessible: No such file or directory.")
```
Comment (greut) [inline cmd/exo/cmd/ssh.go]: what is not found?

Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/ssh.go]: image

Comment (greut) [reply cmd/exo/cmd/ssh.go]: where wasn't the what not found? :monkey_face:

**Comment by greut** [comment]
@pierre-emmanuelJ what do you think?

**Pull request merged**